src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number) {
+      operation(number);
```

Comment:
The `operation` method is calling a function passed as an argument but it doesn't use the `this` keyword. It should be `this.operation(number)` to call the operation on the current calculator instance.